weaviate/gql/filter.py

```diff
@@ -423,6 +423,161 @@ def __str__(self):
         return near_image + "} "
 
 
+class NearMedia(Filter):
+    def __init__(
+        self,
+        content: dict,
+        media_type: str,
```

Comment:
I know the other filters have the same signature, but this is really hard to use.

Would it make sense to:
- use an ENum for media-type
- have the content of the dict as parameters? eg 
```
    def __init__(
        self,
        media_type: MediaType,  <=default value needed?
       media_type2: MediaType. <= is this suppoesd to be there two times?
         certainty: float
        distance: float
       
```

Reply:
Yes, that's a great point, I have added `MediaType` enum definition